sdf_test/objects_list_test.yaml

```diff
@@ -1,7 +1,7 @@
-- {id: "coke-1", type: "coke_can", x: 3.196, y: 4.652, z: 0.87, roll: 0, pitch: 1.57, yaw: 0}
-- {id: "coke-2", type: "coke_can", x: 3.214, y: 4.050, z: 0.87, roll: 0, pitch: 0, yaw: 0.2}
-- {id: "coke-3", type: "coke_can", x: 3.463, y: 4.353, z: 0.87, roll: 0, pitch: 0, yaw: 0.4}
-- {id: "coke-4", type: "coke_can", x: 1.056, y: 3.471, z: 0.8 , roll: 0, pitch: 0, yaw: 0.6}
-- {id: "coke-5", type: "coke_can", x: 0.676, y: 3.559, z: 0.8 , roll: 0, pitch: 0, yaw: 0.8}
```

Comment:
These are round, so we will not see that. But good to test it. Use other objects if possible.